Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.py

```diff
@@ -64,31 +66,54 @@ def get_params_port(params: dict = demisto.params()) -> int:
     return port
 
 
-def refresh_outbound_context(indicator_query: str, out_format: str, limit: int = 0) -> str:
+def refresh_outbound_context(indicator_query: str, out_format: str, limit: int = 0, offset: int = 0) -> str:
     """
     Refresh the cache values and format using an indicator_query to call demisto.searchIndicators
     Returns: List(IoCs in output format)
     """
     now = datetime.now()
-    iocs = find_indicators_with_limit(indicator_query, limit)  # poll indicators into list from demisto
+    iocs = find_indicators_with_limit(indicator_query, limit, offset)  # poll indicators into list from demisto
     out_dict = create_values_out_dict(iocs, out_format)
     out_dict[CTX_MIMETYPE_KEY] = 'application/json' if out_format == FORMAT_JSON else 'text/plain'
-    save_context(now, out_dict)
+    save_context(now, limit, offset, out_format, indicator_query, iocs, out_dict)
     return out_dict[CTX_VALUES_KEY]
 
 
-def save_context(now: datetime, out_dict: dict):
+def save_context(now: datetime, limit: int, offset: int, out_format: str, query: str, iocs: list, out_dict: dict):
```

Comment:
This function is pretty redundant, as it only contains a single line of code, which calls  `demisto.setIntegrationContext`. Consider removing the function.